cmd/installer/pkg/install/manifest.go

```diff
@@ -130,6 +130,10 @@ func (m *Manifest) ExecuteManifest() (err error) {
 			}
 		}
 
+		if err = bd.RereadPartitionTable(); err != nil {
+			return fmt.Errorf("failed to re-read partition table on %q: %w", dev, err)
+		}
+
```

Comment:
There is an implicit reread (slightly different syscall) when adding a partition. This error, I think, is due to a race condition between parallel builds. They all share the same loopback pool.

Reply:
I checked it multiple times and saw the failure with no other builds running, that's why I thought it's something else. 

On other hand, it always fails straight after partitioning before formatting, so it should be something there

Reply:
In that case, then yeah that is strange. Just for reference, here is the other syscall we use to inform the kernel of the newly added partition: https://github.com/talos-systems/talos/blob/master/pkg/blockdevice/table/gpt/gpt.go#L315-L317

Reply:
for example: https://ci.dev.talos-systems.io/talos-systems/talos/9619/1/18

all the failures seem to be exactly the same

Reply:
and yep, `InformKernelOfAdd` should have took care of that :shrug:

Reply:
do you think it might cause issues in real installs? I can make it ignore the error and just print the warning if it fails to re-read the partition table?

Reply:
updated failure to be a warning, so it should be harmless to add